orangecontrib/timeseries/widgets/owspiralogram.py

```diff
@@ -261,6 +256,9 @@ def _enum_str(enum_value, inverse=False):
     return enum_value.name.replace('_', ' ').lower()
 
 
+DEFAULT_AGG_FUNC = list(AGG_FUNCTIONS)[0]
```

Comment:
If you keep `AGG_FUNCTIONS`, as described above, you won't need to cast to list.

Otherwise, `next(iter(x))` is more efficient than `list(x)[0]` (not that it matters here).